InformationScripting/src/queries/SubstractNodesOperator.cpp

```diff
@@ -33,6 +35,8 @@ QList<TupleSet> SubstractNodesOperator::execute(QList<TupleSet> input)
 	Q_ASSERT(input.size() == 2);
 	auto setA = input[0];
 	auto setB = input[1];
+	// TODO: this is not the correct place to do this. But where is it??
```

Comment:
Hmm, I'm not entirely convinced what's the correct operation of the subtract operator. Strictly speaking, you shouldn't extend the B set before subtracting it from A. But on the other hand, it might be useful, in which case doing it here seems OK, why is it not OK? Maybe it should be an option. I can also imagine that there might be a more complex conditions that determines whether we should subtract something.